### PR TITLE
Up19 106 migrate pages

### PR DESCRIPTION
**Pull request by StrikeNP**

### CODE REVIEW

File: src/apiKeys.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (murphyad) [inline src/apiKeys.json]: This file should be in the .gitignore.  We don't want to store our api keys on the repo for anyone to see.

File: src/pages/Organizations/organizationRegistration.tsx [hunk 2]
```diff
@@ -284,42 +284,42 @@ class OrganizationLogoField extends React.Component {
...
+//   onImageSelected = (event) => {
+//     var file = event.target.files[0];
+//     let reader = new FileReader();
+//     reader.readAsDataURL(file);
+//     reader.onload = () => {
+//       this.setState({path: reader.result});
+//       this.props.saveOrgBanner(this.state.valid ? reader.result : undefined);
+//       console.log(this.state.value);
+//     };
+//     reader.onerror = (error) => {
```
Comment (murphyad) [inline src/pages/Organizations/organizationRegistration.tsx:303]: Do we want to remove these comments?

**Review by murphyad — APPROVED** [review]
I left a few comments for you to address. Other than that this is good to go!

File: src/pages/Header/Header.js [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+//     let OrgNameByID = getOrganizationById(({organization, loading, error}) => {
+//         if (loading) {
+//             return (<strong>Loading...</strong>);
+//         } else if (error) {
+//             console.log(error);
+//             return (<p style={{color: "#F00"}}>API error</p>);
+//         } else {
+//             if (organizationId == organization.id) currentOrganization = organization;
+//             let linkArray = window.location.href.split("/");
+//             let link = (linkArray.length > 2 ? linkArray.slice(3).join("/") : "");
```
Comment (DonalMoloneyMSOE) [inline src/pages/Header/Header.js]: Maybe this should be deleted if its is commented out.

**Review by DonalMoloneyMSOE — APPROVED** [review]

**Pull request merged**